syntax = "proto3";

message HelloRequest {
    string name=1;
}

message HelloResponse {
    string name=1;
    string message=2;
}

service Greeter {
    rpc PrintHelloWorld(HelloRequest) returns (HelloResponse) {}
    rpc PrintAgain(HelloRequest) returns (HelloResponse) {}
}